src/realm/sync/client.cpp

```diff
@@ -1372,6 +1374,7 @@ void SessionWrapper::finalize()
     ClientImpl::Connection& conn = m_sess->get_connection();
     conn.initiate_session_deactivation(m_sess); // Throws
 
+    m_flx_pending_bootstrap_store = nullptr;
```

Comment:
I was making sure the `m_flx_pending_bootstrap_store` didn't stick around after `m_sess` was deleted since the pending bootstrap store gets its logger reference from there - technically it doesn't matter, but just to preserve order of creation/deletion with the logger in this case.
A new pending bootstrap store object will be created just after `m_sess` is instantiated.

Reply:
Can you add a comment here about why we're releasing this here? And if you want you can just call `m_flx_pending_bootstrap_store.reset()`

Reply:
Will do